### ISSUE TITLE
The Gradle Language Server server crashed 5 times in the last 3 minutes. The server will not be restarted.

### ISSUE THREAD

**Issue by AHilyard** [badsyntax/vscode-gradle#1024]
Extension Name: vscode-gradle
Extension Version: 3.8.0
OS Version: Windows 10
VSCode version: 1.60.2

**Describe the bug**
As the title states, the Gradle language server crashes immediately when starting VSCode.  The Gradle Language Server output tab shows the following stacktrace (five times):
```
Exception in thread "main" java.lang.NoSuchMethodError: java.util.List.of(Ljava/lang/Object;Ljava/lang/Object;)Ljava/util/List;
	at com.microsoft.gradle.GradleServices.<clinit>(GradleServices.java:86)
	at com.microsoft.gradle.GradleLanguageServer.<init>(GradleLanguageServer.java:76)
	at com.microsoft.gradle.GradleLanguageServer.main(GradleLanguageServer.java:54)
[Info  - 9:56:05 AM] Connection to server got closed. Server will restart.
```
This didn't start happening until this morning (it appears the extension was updated last night).

**To Reproduce**
Install extension, restart VSCode.

**Expected behavior**
I would expect the Gradle Language Server to start properly.

**Screenshots**

![Screenshot 2021-09-29 100829](https://user-images.githubusercontent.com/12298785/135316664-e6bff8f0-eec4-43cd-ad7e-cee1e88626d6.png)
![Screenshot 2021-09-29 100853](https://user-images.githubusercontent.com/12298785/135316666-ace54a3a-1015-4cb7-baff-368a37776bc1.png)
![Screenshot 2021-09-29 100952](https://user-images.githubusercontent.com/12298785/135316668-59914989-ae2d-46d6-bd1e-e215d2f22c24.png)

**Output from "Gradle Tasks"**
```
[info] [gradle-server] Server started, listening on 60802
[info] Gradle client connected to server
```

**Does the bug still exist if you disable all other extensions?**
Yes

**Additional context**
I am using a gradle wrapper with gradle version 4.10.3.

**Comment by AHilyard** [comment]
**Discovered the issue:** I have several different versions of Java installed for different projects and I was explicitly setting the extension's Java home to Java8.  This worked with previous versions of the extension, but no longer does as of 3.8.0.

If anyone has this issue, it can be remedied by updating the extension's java version to at least 9.  ~~I am using 16 now.~~
_Edit for future readers: Java 16 actually does not work for Gradle 4.10.3, so as of vscode-gradle 3.8.0, certain older versions of Gradle **do not work**._

**Comment by CsCherrYY** [comment]
@AHilyard Your workaround can resolve this issue, but there are still some bugs for Java8 users :(. (As we know, `List.of()` becomes available since Java9), I'll fix it.

**Comment by aishfenton** [comment]
Seeing the issue too. But in our case, we can't move off Java8 any time soon. So a fix would be great.

**Comment by CsCherrYY** [comment]
the fix will be available in 3.8.1
